ivy/data_classes/container/experimental/statistical.py

```diff
@@ -444,6 +444,130 @@ def nanmean(
             self, axis=axis, keepdims=keepdims, dtype=dtype, out=out
         )
 
+    @staticmethod
+    def static_nanmin(
```

Comment:
The container method should be private (`_static_nanmin`) like `_static_add` in `container/elementwise.py`

Reply:
@vedpatwardhan In the container file the other functions are not private like nanmean,nanprod etc. I mean why only this nanmin function should be private? In the container/elementwise.py all the functions are private.

Reply:
ideally they should be private as well, it must've gotten added as public by mistake. In general all container static methods are supposed to be private 😄

Reply:
> ideally they should be private as well, it must've gotten added as public by mistake. In general all container static methods are supposed to be private 😄

When I made it private I got some error regarding it. I dont remember the exact error but when I removed it all the test cases passed.

Reply:
@vedpatwardhan AttributeError: 'ivy.data_classes.array.array' object has no attribute 'static_nanmin'
ivy/data_classes/container/base.py:4023: AttributeError
this is the error

Reply:
> @vedpatwardhan AttributeError: 'ivy.data_classes.array.array' object has no attribute 'static_nanmin' ivy/data_classes/container/base.py:4023: AttributeError this is the error

kindly ignore fixed it. The container is made private in the latest commit